### DISCUSSION TITLE
Cannot update the parent record when child has onUpdate: Restrict

### DISCUSSION THREAD

**Discussion by morgs32** [prisma/prisma#12881] [Q&A] [answered]
I expect this to work given the documentation [here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/referential-actions#example-usage-1)

The example schema:
```
model Post {
  id       Int    @id @default(autoincrement())
  title    String
  author   User   @relation(fields: [authorId], references: [id], onDelete: Restrict, onUpdate: Restrict)
  authorId Int
}

model User {
  id    Int    @id @default(autoincrement())
  posts Post[]
}
```

> Result of using Restrict:
Users with posts cannot be deleted. The User's id cannot be changed.

**Comment by austincrim** [comment, marked as answer]
Hey @morgs32 👋 ,

Could you clarify a few things for me:
- what code are you trying to run?
- do you get an error message you can share with me?
- when you say "parent" and "child", are you referring to the `User` as the parent and the `Post` as the child?

Thanks!

**Reply by vladi-stevanovic** [reply]
👋 Hello @morgs32  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Reply by morgs32** [reply]
Yikes this looks half-baked to me. Because that sounds like expected behavior. I should have attached the error at least. Shameful! Please delete or archive this if you can - I'm a happy prisma user with no issues at the moment!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
